api/src/main/java/jakarta/persistence/EntityAgent.java

```diff
@@ -0,0 +1,262 @@
+/*
+ * Copyright (c) 2008, 2025 Oracle and/or its affiliates. All rights reserved.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0,
+ * or the Eclipse Distribution License v. 1.0 which is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR BSD-3-Clause
+ */
+
+// Contributors:
+//     Gavin King      - 4.0
+
+package jakarta.persistence;
+
+import java.util.List;
+
+/**
+ * Interface enabling more direct control over interaction with the
+ * database than what is possible with {@link EntityManager}.
+ *
+ * <p>An instance of {@code EntityAgent} must be obtained from
+ * an {@link EntityManagerFactory}, and is able to manage
+ * persistence of entities belonging to the associated persistence
+ * unit.
```

Comment:
> manage persistence

maybe it would be better to say something like: perform operation on entities beloning to ...
just wondering if we say "manage persistance" wouldn't it be brining us closer to a stateful EM?